src/ElfReader.cc

```diff
@@ -268,6 +303,23 @@ bool ElfFileReader::read(size_t offset, size_t size, void* buf) {
   return read_all(fd, offset, size, buf);
 }
 
+ScopedFd ElfFileReader::open_debug_file(const std::string& elf_file_name) {
+  if (elf_file_name.empty() || elf_file_name[0] != '/') {
```

Comment:
Hmm.  This comment became outdated and now its back.

Without a full path, we don't know where to search under /usr/lib/debug.